adapters/repos/db/index.go

```diff
@@ -1980,6 +1980,11 @@ func (i *Index) loadLocalShard(ctx context.Context, shardName string) error {
 	return i.initLocalShardWithForcedLoading(ctx, i.getClass(), shardName, true)
 }
 
+// TODO temp for testing
+func (i *Index) LoadShard(ctx context.Context, shardName string) error {
+	return i.loadLocalShard(ctx, shardName)
```

Comment:
What I'd like this method to do is, "Given that the shard replica's files exist on disk (because they were copied from the other node), have this shard load the data from disk and prepare to receive writes", is there an existing method that I should use/make public? If not, is adding this `LoadShard` which calls `loadLocalShard` the right way to do this? Any other ideas? Do I need to take any locks from the caller (or add a `release` return func)?

@donomii @aliszka @jeroiraz

Reply:
It's already there in adapters/repos/db/shard_lazyloader.go: func (l *LazyLoadShard) Load(ctx context.Context) error

Reply:
Ok, thanks @donomii I'll make `loadLocalShard` public, I appreciate it!